### YamlMime:ManagedReference
items:
- uid: System.DuplicateWaitObjectException
  id: DuplicateWaitObjectException
  children:
  - System.DuplicateWaitObjectException.#ctor
  - System.DuplicateWaitObjectException.#ctor(System.String)
  - System.DuplicateWaitObjectException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.DuplicateWaitObjectException.#ctor(System.String,System.Exception)
  - System.DuplicateWaitObjectException.#ctor(System.String,System.String)
  langs:
  - csharp
  name: DuplicateWaitObjectException
  nameWithType: DuplicateWaitObjectException
  fullName: System.DuplicateWaitObjectException
  type: Class
  summary: "Wyjątek zgłaszany, gdy obiekt występuje więcej niż raz w tablicy obiektów synchronizacji."
  remarks: "Środowisko uruchomieniowe języka wspólnego zapewnia mechanizm synchronizacji wątku oparte na oczekiwanie na wykonanie w tablicy obiektów synchronizacji <xref:System.Threading.WaitHandle>obiektów.</xref:System.Threading.WaitHandle> Jeśli tablica <xref:System.Threading.WaitHandle>obiektów przekazany do <xref:System.Threading.WaitHandle.WaitAll%2A>lub <xref:System.Threading.WaitHandle.WaitAny%2A>zawiera wszystkie zduplikowane systemu operacyjnego duplicatewaitobjectexception — dojść, jest zgłaszany.</xref:System.Threading.WaitHandle.WaitAny%2A> </xref:System.Threading.WaitHandle.WaitAll%2A> </xref:System.Threading.WaitHandle> Aby uzyskać więcej informacji zobacz <xref:System.Threading.WaitHandle>.</xref:System.Threading.WaitHandle>       Duplicatewaitobjectexception — używa COR_E_DUPLICATEWAITOBJECT HRESULT, która ma wartość 0x80131529.       Aby uzyskać listę początkowe wartości właściwości dla wystąpienia duplicatewaitobjectexception — zobacz <xref:System.DuplicateWaitObjectException.%23ctor%2A>konstruktorów.</xref:System.DuplicateWaitObjectException.%23ctor%2A>      > [!NOTE] > Tego wyjątku nie jest uwzględniony w [.NET dla Sklepu Windows apps](http://go.microsoft.com/fwlink/?LinkID=247912) lub, ale jest generowany przez niektóre elementy członkowskie, które są. Aby w takim przypadku catch wyjątku, napisz `catch` instrukcji dla <xref:System.ArgumentException>zamiast.</xref:System.ArgumentException>"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class DuplicateWaitObjectException : ArgumentException
  inheritance:
  - System.ArgumentException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.DuplicateWaitObjectException.#ctor
  id: '#ctor'
  parent: System.DuplicateWaitObjectException
  langs:
  - csharp
  name: DuplicateWaitObjectException()
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException()
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.DuplicateWaitObjectException&quot;> </xref> klasy."
  remarks: "Ten konstruktor inicjuje <xref:System.Exception.Message%2A>Właściwość nowego wystąpienia na komunikat dostarczany przez system, który opisuje błąd, takie jak &quot;Zduplikowane obiekty w argumencie&quot;.</xref:System.Exception.Message%2A> Ten komunikat bierze pod uwagę bieżące ustawienia kulturowe systemu.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.DuplicateWaitObjectException>.</xref:System.DuplicateWaitObjectException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Odwołanie o wartości null (`Nothing` w języku Visual Basic). |   | <xref:System.ArgumentException.Message%2A>| Ciąg z komunikatem o zlokalizowane. |</xref:System.ArgumentException.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DuplicateWaitObjectException ();
    parameters: []
  overload: System.DuplicateWaitObjectException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.DuplicateWaitObjectException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.DuplicateWaitObjectException
  langs:
  - csharp
  name: DuplicateWaitObjectException(String)
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException(String)
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.DuplicateWaitObjectException&quot;> </xref> klasa o nazwie parametru, który powoduje, że ten wyjątek."
  remarks: "Zawartość `parameterName` ma na celu rozumiane przez człowieka.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.DuplicateWaitObjectException>.</xref:System.DuplicateWaitObjectException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Odwołanie o wartości null (`Nothing` w języku Visual Basic). |   | <xref:System.ArgumentException.Message%2A>| Ciąg z komunikatem o zlokalizowane. |   | <xref:System.ArgumentException.ParamName%2A>| Ciąg nazwy parametru. |</xref:System.ArgumentException.ParamName%2A> </xref:System.ArgumentException.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DuplicateWaitObjectException (string parameterName);
    parameters:
    - id: parameterName
      type: System.String
      description: "Nazwa parametru, który spowodował wyjątek."
  overload: System.DuplicateWaitObjectException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.DuplicateWaitObjectException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.DuplicateWaitObjectException
  langs:
  - csharp
  name: DuplicateWaitObjectException(SerializationInfo,StreamingContext)
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException(SerializationInfo,StreamingContext)
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.DuplicateWaitObjectException&quot;> </xref> z zserializowanymi danymi."
  remarks: "Ten konstruktor jest wywoływana podczas deserializacji do odtworzenia obiekt wyjątku przesyłane przez strumień. Aby uzyskać więcej informacji, zobacz [XML i serializacji SOAP](~/add/includes/ajax-current-ext-md.md)."
  syntax:
    content: protected DuplicateWaitObjectException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Obiekt zawierający dane zserializowanego obiektu."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Informacje kontekstowe dotyczące źródła lub miejsca docelowego."
  overload: System.DuplicateWaitObjectException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.DuplicateWaitObjectException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.DuplicateWaitObjectException
  langs:
  - csharp
  name: DuplicateWaitObjectException(String,Exception)
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException(String,Exception)
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.DuplicateWaitObjectException&quot;> </xref> z określonego komunikatu o błędzie i odwołania do wyjątek wewnętrzny będący przyczyną tego wyjątku."
  remarks: "Wyjątek zgłaszany bezpośrednio w wyniku poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A>Właściwości.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Właściwość zwraca tę samą wartość, która została przekazana do tego konstruktora.</xref:System.Exception.InnerException%2A>       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.DuplicateWaitObjectException>.</xref:System.DuplicateWaitObjectException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Wyjątek określony przez `innerException`. |   | <xref:System.ArgumentException.Message%2A>| Ciąg z komunikatem o określonym przez `message`. |</xref:System.ArgumentException.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DuplicateWaitObjectException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
    - id: innerException
      type: System.Exception
      description: "Wyjątek będący przyczyną bieżącego wyjątku. Jeśli `innerException` parametr nie jest <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, bieżący wyjątek jest zgłaszany w <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloku, który obsługuje wyjątek wewnętrzny."
  overload: System.DuplicateWaitObjectException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.DuplicateWaitObjectException.#ctor(System.String,System.String)
  id: '#ctor(System.String,System.String)'
  parent: System.DuplicateWaitObjectException
  langs:
  - csharp
  name: DuplicateWaitObjectException(String,String)
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException(String,String)
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException(String,String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.DuplicateWaitObjectException&quot;> </xref> z określonego komunikatu o błędzie i nazwę parametru, który powoduje, że ten wyjątek."
  remarks: "Zawartość `parameterName` i `message` parametry mają rozumiane przez człowieka. Element wywołujący tego konstruktora jest wymagana, aby upewnić się, że te ciągi zostały zlokalizowane dla bieżącego ustawienia kulturowego systemu.       Wyjątek zgłaszany bezpośrednio w wyniku poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A>Właściwości.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Właściwość zwraca tę samą wartość, która została przekazana do konstruktora lub odwołanie o wartości null (`Nothing` w języku Visual Basic) Jeśli <xref:System.Exception.InnerException%2A>Właściwości nie dostarcza do konstruktora wartości wyjątku wewnętrznego.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.DuplicateWaitObjectException>.</xref:System.DuplicateWaitObjectException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Odwołanie o wartości null (`Nothing` w języku Visual Basic) |   | <xref:System.ArgumentException.Message%2A>| Ciąg z komunikatem o. |   | <xref:System.ArgumentException.ParamName%2A>| Ciąg nazwy parametru. |</xref:System.ArgumentException.ParamName%2A> </xref:System.ArgumentException.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public DuplicateWaitObjectException (string parameterName, string message);
    parameters:
    - id: parameterName
      type: System.String
      description: "Nazwa parametru, który spowodował wyjątek."
    - id: message
      type: System.String
      description: "Komunikat opisujący błąd."
  overload: System.DuplicateWaitObjectException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.ArgumentException
  isExternal: false
  name: System.ArgumentException
- uid: System.DuplicateWaitObjectException.#ctor
  parent: System.DuplicateWaitObjectException
  isExternal: false
  name: DuplicateWaitObjectException()
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException()
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException()
- uid: System.DuplicateWaitObjectException.#ctor(System.String)
  parent: System.DuplicateWaitObjectException
  isExternal: false
  name: DuplicateWaitObjectException(String)
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException(String)
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.DuplicateWaitObjectException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.DuplicateWaitObjectException
  isExternal: false
  name: DuplicateWaitObjectException(SerializationInfo,StreamingContext)
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException(SerializationInfo,StreamingContext)
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.DuplicateWaitObjectException.#ctor(System.String,System.Exception)
  parent: System.DuplicateWaitObjectException
  isExternal: false
  name: DuplicateWaitObjectException(String,Exception)
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException(String,Exception)
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.DuplicateWaitObjectException.#ctor(System.String,System.String)
  parent: System.DuplicateWaitObjectException
  isExternal: false
  name: DuplicateWaitObjectException(String,String)
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException(String,String)
  fullName: System.DuplicateWaitObjectException.DuplicateWaitObjectException(String,String)
- uid: System.DuplicateWaitObjectException.#ctor*
  parent: System.DuplicateWaitObjectException
  isExternal: false
  name: DuplicateWaitObjectException
  nameWithType: DuplicateWaitObjectException.DuplicateWaitObjectException
